docs/troubleshooting.rst

```diff
@@ -6,11 +6,19 @@ Troubleshooting Guide
 Table of Contents
 -----------------
 
-* `I'm stuck in the LightDM login screen after setting up bash-it. <im-stuck-in-the-lightdm-login-screen-after-setting-up-bash-it>`_
+* `I'm stuck in the LightDM login screen after setting up bash-it. <#id1>`_
+* `Pre-commit installation is failed with "unrecognized option --gdwarf-5" error. <#id3>`_ 
```

Comment:
Why is it `#id3` instead of `#id2` ?

Reply:
This is called id attribute scroll/focus. I checked that in restructured text, it is setting the id attribute to `#id1`. Which is wrong in the current https://bash-it.readthedocs.io/en/latest/troubleshooting/im-stuck-in-the-lightdm-login-screen-after-setting-up-bash-it redirected to 404 whereas https://bash-it.readthedocs.io/en/latest/troubleshooting/#id1 takes to the correct heading on the troubleshooting page.

Reply:
These ids' are of the section headers generated by the rst. Even I am wondering why it didn't slugify the header text instead.